atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -509,7 +509,8 @@ def add_log_type(output):
                     'parent_id': parent_id,
                     'repositories': repositories,
                     'tags': list(tags),
-                    'config': config
+                    'config': config,
+                    'autorebuild': is_rebuild(self.workflow)
```

Comment:
Alternative is `build.extra.image.autorebuild`. I'm not entirely sure what we're free to change.